tests/snuba/api/endpoints/test_organization_tagkey_values.py

```diff
@@ -46,7 +47,9 @@ def test_simple(self):
 
         response = self.client.get(url, format='json')
         assert response.status_code == 200, response.content
-        assert response.data == [{'count': 2, 'value': 'orange'}, {'count': 1, 'value': 'apple'}]
+
+        assert [(val['value'], val['count'])
```

Comment:
Are there new keys in the result now that required changing this test assertion?

Reply:
yeah, it's using this serializer now: https://github.com/getsentry/sentry/blob/master/src/sentry/tagstore/types.py#L98

I made the assertion just pull out value / count because it also has timestamps that will change unless i use absolute dates when creating the events